kotsadm/pkg/airgap/airgap.go

```diff
@@ -229,6 +230,11 @@ func CreateAppFromAirgap(pendingApp *types.PendingApp, airgapPath string, regist
 		return errors.Wrap(err, "failed to load kotskinds from path")
 	}
 
+	err = supportbundle.CreateRenderedSpec(a.ID, a.CurrentSequence, "", true, kotsKinds.SupportBundle)
```

Comment:
just want to note that `a.CurrentSequence` is _not_ the deployed sequence which is probably what you're looking for here.